addons/binding/org.openhab.binding.dscalarm/OSGI-INF/DSCAlarmBridgeDiscovery.xml

```diff
@@ -16,5 +16,7 @@
    <service>
       <provide interface="org.eclipse.smarthome.config.discovery.DiscoveryService"/>
    </service>
+   <reference bind="setThingRegistry" cardinality="1..1" interface="org.eclipse.smarthome.core.thing.ThingRegistry" name="ThingRegistry" policy="static" unbind="unsetThingRegistry"/>
```

Comment:
This looks fishy to me - a binding should never require access to the thing registry or the thing type registry.
Or am I missing something important here?